src/ZoneServer/World/Actors/Monsters/Mob.cs

```diff
@@ -307,11 +316,31 @@ public void Kill(ICombatEntity killer)
 
 			this.DisappearTime = DateTime.Now.AddSeconds(2);
 
-			if (killer is Character characterKiller)
+			// If I have a master, remove my variables from my master so I can be resummoned
```

Comment:
Minor thing, but I believe we don't use a class' POV to document its behavior anywhere else in the code, making it feel a little out of place.

Reply:
This is fixed.